service/src/main/kotlin/fi/espoo/evaka/pis/CreatorsAndModifiers.kt

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.pis
+
+import fi.espoo.evaka.application.ApplicationType
+import fi.espoo.evaka.shared.ApplicationId
+import fi.espoo.evaka.shared.EvakaUserId
+import fi.espoo.evaka.shared.db.DatabaseEnum
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+
+enum class CreateSource : DatabaseEnum {
+    USER,
+    APPLICATION,
+    DVV;
+
+    override val sqlType = "create_source"
+}
+
+sealed class Creator {
```

Comment:
I'm happy with "creator" but maybe "origin" or "source" would have been more common terms for this

Reply:
There's also Modifier which relates to modified_at/by while Creator relates to created_at/by, so I think this is at least consistent, even if a bit weird sounding otherwise.